src/main/application/schemas/doc.sd

```diff
@@ -82,23 +82,24 @@ schema doc {
 
     rank-profile documentation inherits default {
         rank-properties {
-            $titleWeight: 2.0
-            $contentsWeight: 1.0
-            $headerWeight: 1.0
-            $pathWeight: 1.0
+            query(titleWeight): 2.0
+            query(contentsWeight): 1.0
```

Comment:
```suggestion
            query(contentWeight): 1.0
```

Reply:
To be aligned with the name of the field and what is used in another rank profile. Also the expression and summary-features must be changed.

Reply:
good point, will do